modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -10,15 +10,19 @@ After you provision the adapter, you must create an API to bind to the service.
 +
 If you want to create REST or SOAP APIs with OAS or WSDL definition, use Anypoint Platform.
 +
-Note that if you want to upload a REST API or OAS asset type, use a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, use a .wsdl or .xml file.
-* <<create_api-CRD, Create an API using Anypoint Service Mesh Autocreate>>
+If you want to upload a REST API or OAS asset type, you must use a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, use a .wsdl or .xml file.
```

Comment:
Same comment as before to remove "you must."

Reply:
Done! Thanks!